src/routes/api/user.routes.js

```diff
@@ -46,13 +51,17 @@ router.post(
 
 router.post(
   '/login',
+  validateLogin,
   getUserByEmail,
-  checkUserVerified,
   checkDisabledAccount,
-  CheckLoginPassword,  
+  checkUserVerified,
+  CheckLoginPassword,
+  checkSeller,
   UserController.loginUser
 );
 
+router.post('/login/verify/:id', validateAuthCode, checkIfUserExistById, verifyAuthCode, removeAuthCode, UserController.loginUser);
```

Comment:
Replace `/:id` with `/:userId`

Reply:
Here, when I use userId, that middleware, of checkUserexistsbyid, fails, since it is looking for the id param instead of the userId param. Should I change it too?